### DISCUSSION TITLE
A lot of 1KB, essentially nonexistent cover art?

### DISCUSSION THREAD

**Discussion by Kamryx** [RandomNinjaAtk/docker-lidarr-extended#89] [Q&A] [answered]
Pretty much the title, when AMD runs, it seems to have about a 70% success rate on actually downloading an open-able and usable album cover art JPG. Below is what it looks like.

![image](https://user-images.githubusercontent.com/65583898/193440576-fe761451-da1f-4887-bc49-1b539145c0ca.png)
![image](https://user-images.githubusercontent.com/65583898/193440686-6ae262e2-37a0-48f7-8a82-f8afdec4a834.png)

When I try to open on W11: 
![image](https://user-images.githubusercontent.com/65583898/193440599-40d124c0-f8a9-4ddb-b0fe-b7337d51f9ba.png)

Some download and open just fine: 
![image](https://user-images.githubusercontent.com/65583898/193440615-a69c07ac-aa5e-4d2b-92c7-7abbcfabc823.png)

Any idea why this is? Do I have something setup wrong? Expected behavior? Important to note, all of these that I've checked DO have proper and correct cover art on MusicBrainz.

**Comment by RandomNinjaAtk** [comment, marked as answer]
Artwork is handled by Lidarr, not the script, there is a post processing script in lidarr, that will extract embedded artwork that you can disable. But again, this sounds like a Lidarr issue, not a script issue.
